default.nix

```diff
@@ -1,17 +1,35 @@
 let
   sources = import ./nix/sources.nix;
-  pkgs = import sources.nixpkgs {};
+  pkgs = import sources.nixpkgs {
+    config = {};
+    overlays = [
+      (import ./nix/overlay.nix)
+    ];
+  };
+  profileEnv = pkgs.writeTextFile {
+    name = "profile-env";
+    destination = "/.profile";
+    # This gets sourced by direnv. Set NIX_PATH, so `nix-shell` uses the same nixpkgs as here.
+    text = ''
+      export NIX_PATH=nixpkgs=${toString pkgs.path}
+    '';
+  };
 in {
+  inherit profileEnv;
+
   env = pkgs.buildEnv{
     name = "wire-server-deploy";
-    paths = with pkgs; [
-      terraform_0_13
-      python37Packages.poetry
-      awscli
-      sops
-      gnupg
-      yq
-      gnumake
+    paths = [
+      profileEnv
+      pkgs.ansible_2_7_with_libs
+      pkgs.awscli
+      pkgs.gnumake
+      pkgs.gnupg
+      pkgs.kubectl
```

Comment:
hm, this loads kubectl 1.18.8 - and that is unlikely to be compatible with our clusters that are on server version 1.14 (our helm charts are known to work on 1.14 and I think 1.15 - but fail on 1.16 and above currently - see https://github.com/zinfra/backend-issues/issues/1210 and https://github.com/zinfra/backend-issues/issues/1570). Can you provide kubectl 1.14 here?

Reply:
I think kubectl should be fine connecting to old kubernetes clusters. We can override, but I want to first test this.

Reply:
It's definitely not always fine. I burnt myself pretty badly on this before. E.g.   default between versions certain  objects get promoted to different version namespaces (E.g. in 1.18 I think `ingress` is now not in `beta` anymore). This means `kubectl` can't be used to inspect objects by default and leads to weird bugs.  (shows that there are 0 ingresses or deployments, because your deployment was deployed as `v1beta1` instead of as `apps/v1`)

For the stable apis in k8s it's never a problem; but we don't just exclusively use the stable ones.

Reply:
Version skew policies are documented here: https://kubernetes.io/docs/setup/release/version-skew-policy/#kubectl
 1.18 kubectl will not guarantee to work with 1.14

Reply:
Good point. So for now we should probably provide 1.14 here.

However, we don't clearly describe which version of kubernetes we expect in `wire-server-deploy` (only in `wire-docs`, and a bit hand-wavy).

Our kubespray probably only deploys 1.14 because the playbooks default to this version. This should really be more explicit.

Reply:
Done d02ffc3dbcbfe3f5792ab5c1267a7075b2364cc2